run.py

```diff
@@ -32,11 +32,27 @@ def main():
     logging.basicConfig(level=args.log_level)
 
     show_skipped = '--show-skipped' if args.show_skipped else '--no-skipped'
-    tags_arg = f'--tags {args.tags}' if args.tags else ''
+
+    tags_arg = parse_tags(args.tags)
+
     args = f'--logging-level {args.log_level} --format {args.format} {args.feature_directory} {tags_arg} {show_skipped}'
 
     return behave_executable.main(args)
 
 
+# --tags=~@tag1,~@tag2 doesn't work.  Needs to be --tags @tag1 --tags ~@tag2
```

Comment:
From looking at the docs I think that's the correct way to do it. It applies some logic operators on it when you start to specify `,` 
```
Beware that if you want to use several negative tags to exclude several tags
you have to use logical AND::

    --tags ~@fixme --tags ~@buggy.
```

Reply:
yes I think I was getting confused with our call through from Makefile too, then in run.py it rebuilds it.  
Not as part of this ticket, but we need might want to tweak it longterm - but as requied

Reply:
I'm going to follow up with a little ticket to remove the `run.py`, it's only really serving to hold some default behave config which can be done in other ways, and it makes passing through args like this more awkward.